apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -216,7 +216,7 @@ function ListBoxInline({ options, layout }) {
     }
   }, [searchContainer && searchContainer.current, showSearch, search, focusSearch]);
 
-  const { wildCardSearch, searchEnabled, layoutOptions = {} } = layout ?? {};
+  const { wildCardSearch, searchEnabled, autoConfirm = false, layoutOptions = {} } = layout ?? {};
```

Comment:
This new setting should be added to the api spec as well

Reply:
✅